aviary/docs/examples/reserve_missions.ipynb

```diff
@@ -36,14 +88,55 @@
    "outputs": [],
    "source": [
     "# Testing Cell\n",
-    "from aviary.examples.reserve_missions.run_reserve_mission_fixedrange import phase_info\n",
     "from aviary.interface.utils.check_phase_info import check_phase_info, HEIGHT_ENERGY\n",
-    "from aviary.docs.tests.utils import check_value\n",
+    "from aviary.utils.functions import get_path\n",
+    "from aviary.docs.tests.utils import glue_variable, check_value, get_all_keys\n",
+    "# from aviary.docs.tests.utils import check_contains, expected_error\n",
+    "\n",
+    "folder = get_path('examples/reserve_missions')\n",
     "\n",
+    "from aviary.examples.reserve_missions.run_reserve_mission_fixedrange import phase_info\n",
     "check_phase_info(phase_info, HEIGHT_ENERGY);\n",
     "user_opts = phase_info['reserve_cruise']['user_options']\n",
     "check_value(user_opts['target_distance'],(200, 'km'))\n",
-    "check_value(user_opts['reserve'],True)"
+    "check_value(user_opts['reserve'],True)\n",
+    "\n",
+    "run_reserve_mission_fixedtime = 'run_reserve_mission_fixedtime.py'\n",
+    "get_path(folder.joinpath(run_reserve_mission_fixedtime))\n",
+    "glue_variable(str(run_reserve_mission_fixedtime), md_code=True)\n",
+    "\n",
+    "from aviary.examples.reserve_missions.run_reserve_mission_fixedtime import phase_info\n",
+    "check_phase_info(phase_info, HEIGHT_ENERGY);\n",
+    "user_opts = phase_info['reserve_cruise']['user_options']\n",
+    "check_value(user_opts['target_duration'],(30, 'min'))\n",
+    "check_value(user_opts['reserve'],True)\n",
+    "all_keys = get_all_keys(phase_info['reserve_cruise'], track_layers=True)\n",
+    "for key in all_keys:\n",
+    "    glue_variable(key.replace('.','-'), md_code=True)\n",
+    "print(all_keys)\n",
+    "\n",
+    "# ****NOTE**** These were not removed, as the docs seem to indicate\n",
```

Comment:
@Kenneth-T-Moore @ehariton

Reply:
I think I would consider unexpected behavior as a bug, so it would be fine to put up a bug issue and merge this PR as is.